mbed_tools_ci_scripts/generate_docs.py

```diff
@@ -36,13 +39,37 @@ def _generate_pdoc_command_list(output_directory: Path,
     ]
 
 
-def generate_docs(output_directory: Path, module: str) -> int:
-    """Triggers building the documentation."""
+def generate_documentation(output_directory: Path,
+                           module_to_document: str) -> None:
+    """Ensures the documentation is in the correct location.
+
+    Pdoc nests its docs output in a folder with the module's name.
+    This process removes this unwanted folder.
+
+    Raises:
+        Exception if an error happened during the generation.
```

Comment:
Is this true? The pdoc error will be propagated, but I don't think this code raises any exceptions?

Reply:
I think this is far too general an exception to state in the raises clause FWIW. We should only put exceptions that actually relate to the API here.

Reply:
I removed it